src/sentry/grouping/ingest/seer.py

```diff
@@ -187,19 +206,19 @@ def get_seer_similar_issues(
     should go in (if any), or None if no neighbor was near enough.
     """
     event_hash = event.get_primary_hash()
-    stacktrace_string = get_stacktrace_string(get_grouping_info_from_variants(variants))
     exception_type = get_path(event.data, "exception", "values", -1, "type")
 
     request_data: SimilarIssuesEmbeddingsRequest = {
         "event_id": event.event_id,
         "hash": event_hash,
         "project_id": event.project.id,
-        "stacktrace": stacktrace_string,
+        "stacktrace": event.data.get("stacktrace_string", ""),
         "exception_type": filter_null_from_string(exception_type) if exception_type else None,
         "k": num_neighbors,
         "referrer": "ingest",
         "use_reranking": options.get("seer.similarity.ingest.use_reranking"),
     }
+    event.data.pop("stacktrace_string", None)
```

Comment:
We don't need this - only the data in `event.data.data` is stored. This is from the end of `save_error_events`:

https://github.com/getsentry/sentry/blob/2849ca845c7b589faf4573ee9a7abe10622dd886/src/sentry/event_manager.py#L620